src/cdk/stepper/stepper.ts

```diff
@@ -185,13 +185,13 @@ export class CdkStepper implements OnDestroy {
       if (this._anyControlsInvalidOrPending(index) || index < this._selectedIndex &&
           !this._steps.toArray()[index].editable) {
         // remove focus from clicked step header if the step is not able to be selected
-        this._stepHeader.toArray()[index].nativeElement.blur();
+        this._stepHeader.toArray()[index].blur();
```

Comment:
Actually, looking at why we needed to call `blur` in the first place brought me here, and this seems wrong. We shouldn't just clear the focus, we should set `tabindex=-1` ahead of time, or if that's not possible maybe cancel the `mousedown` event. That way the user doesn't lose their focus position

Reply:
Agreed that we shouldn't blur it, I was primarily going for parity with the old approach. I've gotten rid of the `blur` and it seems like the tabindex was already being set to -1.